### PR TITLE
Adding in url links and style

### PR DESCRIPTION
**Pull request by petermoyer**
Adding in url links and style

- Update the style guides and links

- [ x] My code follows the Great Expectations [style guide](https://docs.greatexpectations.io/en/latest/contributing/style_guide.html?highlight=style%20guide)
- [ x] I have performed a [self-review](https://docs.greatexpectations.io/en/latest/contributing/contribution_checklist.html?highlight=checklist) of my own code
- [x ] I have made corresponding changes to the documentation

Thank you for submitting!

### CODE REVIEW

File: docs/contributing/contributing-checklist.md [hunk 1]
```diff
@@ -4,118 +4,117 @@ title: Contribution Checklist
...
+#### 2. Have you followed the Style Guide for code and comments?
 
-If your PR contains only changes to contrib directory (community contributed Expectations), review your PR against this Acceptance Checklist
+* The [Style Guide](/docs/contributing/contributing-style) is here.
 
-## 3. Is your branch up to date with upstream/develop?
+* Thanks for helping us keep the codebase and documentation clean and consistent, so that it’s easier to maintain it as a community!
 
-Update your local repository with the most recent code from the main Great Expectations repository.
+* If your PR contains only changes to `contrib` directory (community contributed Expectations), review your PR against this Acceptance Checklist
```
Comment (Shinnnyshinshin) [inline docs/contributing/contributing-checklist.md:25]: `Acceptance Checklist` should be a link

File: docs/contributing/contributing-checklist.md [hunk 2]
```diff
@@ -4,118 +4,117 @@ title: Contribution Checklist
...
 
-If your PR contains only changes to contrib directory (community contributed Expectations), please skip this step.
+:::note
+If your PR contains **only** changes to `contrib` directory (community contributed Expectations), please skip this step.
+:::
 
-Please add a bullet point to docs/changelog.rst, in the develop section.
-Please group in the following order: [BREAKING], [FEATURE], [ENHANCEMENT], [BUGFIX], [DOCS], [MAINTENANCE]
+* Please add a bullet point to `docs/changelog.rst`, in the develop section.
+* Please group in the following order:  [FEATURE], [BUGFIX], [DOCS], [MAINTENANCE]
```
Comment (Shinnnyshinshin) [inline docs/contributing/contributing-checklist.md]: question about this point. The current changlog that we use for releases is actually our RTD changelog, which lives in `docs_rtd/changelog.rst`. 

 Maybe we could think about migrating the changelog to the new Netlify site which would turn it into a markdown file (so `docs/changelog.md`) and start using it officially once the Netlify site is "officially" released? Or even before.

File: docs/contributing/contributing-github.md [hunk 3]
```diff
@@ -2,59 +2,63 @@
...
-Start editing
-## 1A. Edit from docs.greatexpecations.io
+### Start editing
+
+#### 1A. Edit from https://knoxpod.netlify.app/docs/
+
+* Go to the [Great Expectations docs](https://knoxpod.netlify.app/docs/).
 
-Go to the Great Expectations docs.
+* On each page, you’ll see an `Edit` button in the lower left. Click this to go to the source file in the Great Expectations GitHub repo.
```
Comment (Shinnnyshinshin) [inline docs/contributing/contributing-github.md:15]: Knitpick that this is `Edit this page`, and the links are currently broken. It seems to add an extra `/docs/` to the link: 

Example: 

`https://github.com/great-expectations/great_expectations/tree/develop/docs/docs/tutorials/getting-started/intro.md`
^ notice the `/docs/docs/`

File: docs/contributing/contributing-misc.md [hunk 4]
```diff
@@ -2,97 +2,102 @@
 title: Contributing Misc and CLA
 ---
```
Comment (Shinnnyshinshin) [inline docs/contributing/contributing-misc.md:4]: Request to add the `Core Team` section. Does not have to be done in this PR.

File: docs/contributing/contributing-setup.md [hunk 5]
```diff
@@ -5,158 +5,154 @@ title: Setting up your Dev Environment
...
 
-Documentation will be generated in docs/build/html/ with the index.html as the index page.
+* Note: we use `autoapi` to generate API reference docs, but it’s not compatible with pandas 1.1.0. You’ll need to have pandas 1.0.5 (or a previous version) installed in order to successfully build docs.
 
-Note: we use autoapi to generate API reference docs, but it’s not compatible with pandas 1.1.0. You’ll need to have pandas 1.0.5 (or a previous version) installed in order to successfully build docs.
+#### Run tests to confirm that everything is working
 
-Run tests to confirm that everything is working
-You can run all tests by running pytest in the great_expectations directory root. Please see Testing for testing options and details.
+* You can run all tests by running `pytest` in the great_expectations directory root. Please see Testing for testing options and details.
```
Comment (Shinnnyshinshin) [inline docs/contributing/contributing-setup.md]: This link to `Testing` should be made to the `Contribution and Testing` doc that you wrote.

File: docs/contributing/contributing.md [hunk 6]
```diff
@@ -6,22 +6,24 @@ Welcome to the Great Expectations project! We’re very glad you want to help ou
 
 Our goal is to make your experience as great as possible. Please follow these steps to contribute:
 
-## 1. Join the community on Slack
-Go to greatexpectations.io/slack and introduce yourself in the #contributors-contributing channel.
+#### 1. Join the community on Slack
 
-## 2. Set up your development environment
-Follow these instructions to set up your dev environment.
+* Go to [greatexpectations.io/slack](https://greatexpectations.io/slack) and introduce yourself in the #contributors-contributing channel.
```
Comment (Shinnnyshinshin) [inline docs/contributing/contributing.md]:
```suggestion
* Go to [greatexpectations.io/slack](https://greatexpectations.io/slack) and introduce yourself in the `#contributors-contributing` channel.
```

File: docs/contributing/contributing-checklist.md [hunk 7]
```diff
@@ -4,118 +4,117 @@ title: Contribution Checklist
 
 Following these instructions helps us make sure the code review and merge process go smoothly.
 
-Before submitting a pull request
-Once your code is ready, please go through the following checklist before submitting a pull request.
-
-## 1. Have you signed the CLA?
+###Before submitting a pull request
```
Comment (Shinnnyshinshin) [inline docs/contributing/contributing-checklist.md]:
```suggestion
### Before submitting a pull request
```

File: docs/contributing/contributing-checklist.md [hunk 8]
```diff
@@ -4,118 +4,117 @@ title: Contribution Checklist
...
+#### 4. Wait for a core team member to approve and merge your PR
 
-Once all checks pass, a Great Expectations team member will approve your PR and merge it.
+* Once all checks pass, a Great Expectations team member will approve your PR and merge it.
 
-GitHub will notify you of comments or a successful merge according to your notification settings.
+* GitHub will notify you of comments or a successful merge according to your notification settings.
 
-## 5. Resolve any issues
+## ## 5. Resolve any issues
```
Comment (Shinnnyshinshin) [inline docs/contributing/contributing-checklist.md]:
```suggestion
#### 5. Resolve any issues
```

File: docs/contributing/contributing-test.md [hunk 9]
```diff
@@ -2,22 +2,23 @@
...
 If you did not configure optional backends for testing, tests against these backends will fail.
 
 You can suppress these tests by adding the following flags:
 
-* --no-postgresql will skip postgres tests
-* --no-spark will skip spark tests
-* --no-sqlalchemy will skip all tests using sqlalchemy (i.e. all database backends)
+* `--no-postgresql` will skip postgres tests
+* `--no-spark` will skip spark tests
+* '--no-sqlalchemy` will skip all tests using sqlalchemy (i.e. all database backends)
```
Comment (Shinnnyshinshin) [inline docs/contributing/contributing-test.md]:
```suggestion
* `--no-sqlalchemy` will skip all tests using sqlalchemy (i.e. all database backends)
```

File: docs/contributing/contributing-setup.md (hunk 5 above)
Comment (Shinnnyshinshin) [reply docs/contributing/contributing-setup.md]:
```suggestion
* You can run all tests by running `pytest` in the great_expectations directory root. Please see [Testing](docs/contributing/contributing-test) for testing options and details.
```

File: docs/contributing/contributing-checklist.md (hunk 1 above)
Comment (Shinnnyshinshin) [inline docs/contributing/contributing-checklist.md:25]: Looking into this further [Acceptance Checklist](https://docs.greatexpectations.io/en/latest/contributing/contrib_pull_request_acceptance_checklist.html#contrib-pull-request-acceptance-checklist) seems to be an entirely different doc.. this doesn't have to be part of this PR, but I'd like to add it to the JIRA that tracks the `How to contribute a new Expectation to Great Expectations` how-to-guide

File: docs/contributing/contributing-checklist.md [hunk 10]
```diff
@@ -4,118 +4,117 @@ title: Contribution Checklist
...
+#### 5. Have you documented all the changes in your PR?
 
-If your PR contains only changes to contrib directory (community contributed Expectations), please skip this step.
+:::note
+If your PR contains **only** changes to `contrib` directory (community contributed Expectations), please skip this step.
+:::
 
-Please add a bullet point to docs/changelog.rst, in the develop section.
-Please group in the following order: [BREAKING], [FEATURE], [ENHANCEMENT], [BUGFIX], [DOCS], [MAINTENANCE]
+* Please add a bullet point to `docs/changelog.rst`, in the develop section.
```
Comment (Shinnnyshinshin) [inline docs/contributing/contributing-checklist.md]:
```suggestion
* Please add a bullet point to `docs_rtd/changelog.rst`, in the develop section.
```

**Review by Shinnnyshinshin — APPROVED** [review]
This looks wonderful. Thank you!

File: docs/contributing/contributing-checklist.md (hunk 7 above)
Comment (petermoyer) [reply docs/contributing/contributing-checklist.md]: Added the space

File: docs/contributing/contributing-checklist.md (hunk 8 above)
Comment (petermoyer) [reply docs/contributing/contributing-checklist.md]: Removed the space

File: docs/contributing/contributing-checklist.md (hunk 2 above)
Comment (petermoyer) [reply docs/contributing/contributing-checklist.md]: Tracked in GDOC-170

File: docs/contributing/contributing-github.md (hunk 3 above)
Comment (petermoyer) [reply docs/contributing/contributing-github.md:15]: GDOC-171 to track

File: docs/contributing/contributing-misc.md (hunk 4 above)
Comment (petermoyer) [reply docs/contributing/contributing-misc.md:4]: I removed this to keep changes for addition/retribution at a minimum. Same content can be found in other company links

**Pull request merged**